server/src/main/java/org/candlepin/resource/EntitlementResource.java

```diff
@@ -200,7 +210,8 @@ public Entitlement getEntitlement(
     @Path("{entitlement_id}")
     public void updateEntitlement(
         @PathParam("entitlement_id") @Verify(Entitlement.class) String id,
-        @ApiParam(name = "update", required = true) Entitlement update) {
+        @ApiParam(name = "update", required = true) EntitlementDTO update) {
+        //TODO Why do we accept a whole Entitlement object here, if we only use the quantity field???
```

Comment:
Probably just a generic method originating from CRUD method specs, but this is a very valid question. If we only ever update the quantity, we'd be better served by dropping this endpoint and adding an updateQuantity to replace it.